Makefile

```diff
@@ -92,6 +92,35 @@ make test-integration:
 	make -C ./tests/integration test
 
 
+# Changelog management
+## We use Towncrier (https://towncrier.readthedocs.io) for changelog management
+
+ENTRY_TEXT ?= 'Use [Semantic Versioning](https://semver.org/spec/v2.0.0.html) for your changelog entry'
+CHANGE_ID ?= 9999
+CHANGE_TYPE ?= changed
+
+## Usage: make add-changelog-entry CHANGE_TYPE=changed
+.PHONY: add-changelog-entry
+add-changelog-entry:
+	towncrier create -c "$(ENTRY_TEXT)" "(CHANGE_ID).(CHANGE_TYPE).txt"
```

Comment:
Any documentation for that? Especially using with `CHANGELOG_ID`?

Reply:
My intent here was for this to be simple. If you call this without any arguments, it should generate an example that is easy to manually edit. I can document the options, but using them is awkward imo.

Reply:
I hoped this will be complex solution, and now it seems does it resolves only conflicting issue, but instead you need to perfom multiple manual steps:

```
1. make 
2. edit filename
3. edit file content
```

I would appreciate it more if we run
```
make add-changelog-entry
```
and it will ask me for PR, type and entry text

Reply:
Do you think that's an improvement? I honestly find it simpler to edit the file vs having to put all the arguments in a single command invocation. You can do `make add-changelog-entry CHANGE_ID=2564 CHANGE_TYPE=fixed ENTRY_TEXT="fix: something important", but I don't really think that's any better.

I could make the default text be the title of the most recent commit, but I don't want to make this excessively magical.

Reply:
I proposed rather interactive script

Reply:
Allright, I can add a script for this. Maybe I can even be clever about default values?

Reply:
I would be great, but even dumb script would be nice feature for me :)

Reply:
I think it's easy enough to put the title of HEAD as the default text. PR number and change type need to be put in by hand though (though I can probably infer something about change type from the title).

Reply:
Ok, added it, can you check it out and let me know if this works for you?